UefiTestingPkg/AuditTests/PagingAudit/UEFI/Dxe/App/DxePagingAuditTestApp.c

```diff
@@ -179,6 +242,44 @@ CanRegionBeRWX (
   return FALSE;
 }
 
+/**
+  Check the page table for Read/Write/Execute regions.
+
+  @param[in] Context            Unit test context
+
+  @retval UNIT_TEST_PASSED      The unit test passed
+  @retval other                 The unit test failed
+
+**/
+UNIT_TEST_STATUS
+EFIAPI
+NoReadWriteExecute (
+  IN UNIT_TEST_CONTEXT  Context
+  )
+{
+  UINTN    Index;
+  BOOLEAN  FoundRWXAddress;
+
+  UT_ASSERT_NOT_NULL (mMap.Entries);
+  UT_ASSERT_NOT_EQUAL (mMap.EntryCount, 0);
+
+  Index           = 0;
+  FoundRWXAddress = FALSE;
+
+  for ( ; Index < mMap.EntryCount; Index++) {
+    if (IsPageExecutable (mMap.Entries[Index].PageEntry) && IsPageReadable (mMap.Entries[Index].PageEntry) && IsPageWritable (mMap.Entries[Index].PageEntry)) {
+      if (!CanRegionBeRWX (mMap.Entries[Index].LinearAddress, mMap.Entries[Index].Length)) {
+        UT_LOG_ERROR ("Memory Range 0x%llx-0x%llx is Read/Write/Execute\n", mMap.Entries[Index].LinearAddress, mMap.Entries[Index].LinearAddress + mMap.Entries[Index].Length);

```

Comment:
nit: consider breaking this into multiple lines to avoid line wrap

Reply:
Updated